web/src/components/profile/getErrorStateForAddressField.ts

```diff
@@ -167,10 +178,21 @@ export const getErrorStateForAddressField = ({
         ? (Config.formation.fields as any)[field].error
         : (Config.formation.fields as any)[field].foreign.errorUS;
 
-    const partialAddressError = fieldWithAssociatedFields({
-      associatedFields: ["addressMunicipality", "addressLine1", "addressLine2"],
-      label: zipCodeErrorLabel,
-    });
+    let partialAddressError;
```

Comment:
this should be revisited, we are introducing two different behaviors based on whether they completed formation or not